src/ROUTES.ts

```diff
@@ -616,6 +616,18 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/taxes/new',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/taxes/new` as const,
     },
+    WORKSPACE_TAX_EDIT: {
+        route: 'settings/workspaces/:policyID/tax/:taxID',
+        getRoute: (policyID: string, taxID: string) => `settings/workspaces/${policyID}/tax/${encodeURI(taxID)}` as const,
+    },
```

Comment:
We should have used `encodeURIComponent` instead of `encodeURI` since `taxID` is not a uri. Also we should have added a decode step (similar to other logic e.g. tag).

`encodeURI` does not encode all special chars e.g. `/` which caused this bug https://github.com/Expensify/App/issues/39614